src/main/java/com/alex/eshop/eshop/dto/CategoryDTO.java

```diff
@@ -0,0 +1,31 @@
+package com.alex.eshop.eshop.dto;
```

Comment:
you have doubled 'eshop' package (folder). Would be good to refactor this - remove one.